docs/code-style-guide.md

```diff
@@ -75,6 +75,16 @@ If you want to know more about our coding style you can take a look at our [clan
   const float SCALE_FACTOR = 2.15;
   float distance = catch_distance * SCALE_FACTOR;
   ```
+* Avoid initializing multiple variables on the same line.
+  ```cpp
+  // Incorrect
+  int x, y, z = 0;
+
+  // Correct
+  int x;
+  int y;
+  int z = 0;
```

Comment:
I think this could confuse some people, could we instead set `x` and `y` equal to 0 as well? (I know it's not technically equivalent, but I think it's more clear)

Reply:
done